coveralls-ruby.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |gem|
   gem.required_ruby_version = '>= 1.8.7'
 
   gem.add_dependency 'json', '>= 1.8', '< 3'
-  gem.add_dependency 'simplecov', '~> 0.16.1'
+  gem.add_dependency 'simplecov', '>= 0.16.1'
```

Comment:
This is dangerous change: it allows versions like `0.17`, `0.18`, `0.99`, etc., which (due to 0-major version, which means "alpha") can contain breaking changes.

This should be locked or with `~> 0.17.0`, either via `'>= 0.16.1', '< 0.18'` if you want to support multiple versions.